data/field_meta.go

```diff
@@ -0,0 +1,32 @@
+package data
+
+// FieldMeta attaches structural metadata to each frame
+type FieldMeta struct {
+	// StepSize is the interval between each value.
+	// If no values exist in the values array, this indictes mising data for that interval
+	// Only supports non-zero step sizes
+	StepSize float64 `json:"stepSize,omitempty"`
```

Comment:
1. is `float64` seconds the way to go? i understand Prometheus does it, but wouldn't nanosecond-int64 be more generic? not sure.
2. i'd like to emphasize that "StepSize" should mean: "if there is a time-gap between datapoints larger than StepSize, that is missing data, in the graph do not connect the dots". it should not mean "all timestamps are aligned at `start + integer * StepSize`). (again, i understand in Prometheus it is, but i think a weaker guarantee is more useful)

Reply:
(this is the least important).. we should decide if `stepSize` or `interval` or sometihng else.. most of Grafana says `interval` in the dashboards, right?

Reply:
> is float64 seconds the way to go?

The unit should be based on the field type/units.  For time.Time, we do need to pick something -- I think ms makes the most sense because that is what the frontend will see.  Given that it is a floating point number, the resolution can be more then ms 


Interval vs Step -- I don't feel strongly.  Step works better for non-time sequences, and make sense with log scale also.  "Interval" feels more time specific